src/main/__test__/shell-session.test.ts

```diff
@@ -38,3 +39,24 @@ describe("clearKubeconfigEnvVars tests", () => {
     expect(clearKubeconfigEnvVars({ a: 1, kubeconfig: "1", kUbeconfig: "1" })).toStrictEqual({ a: 1 });
   });
 });
+
+describe("resolveEnv tests", () => {
```

Comment:
Some extra test cases that I think we should cover:

1. `~/foo` should resolve to be the same as `$HOME/foo`. I would use our `resolvePath` function in `utils/paths`


I think for a first pass this should be good enough. Maybe in the future we will support `${foo:-default}` but for the time being this is good.

Reply:
I like that, will update.

Reply:
I tried using `resolvePath`, but it seems to prepend the result with the CWD whenever the path is not absolute, which can occur when var expansion fails.

```
resolveEnv tests › should not resolve invalid var (starts with digit)
    Expected: "$0HOME"
    Received: "C:\\dev\\github.com\\devodev\\lens\\$0HOME"
```

I assume this to fail the pathExists test though, so might be enough?